modules/apps/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/servlet/taglib/JSLoaderTopHeadDynamicInclude.java

```diff
@@ -140,7 +140,7 @@ private String _getURL(
 			return url + StringPool.AMPERSAND;
 		}
 
-		return themeDisplay.getCDNBaseURL();
+		return themeDisplay.getCDNBaseURL() + themeDisplay.getPathContext();
```

Comment:
I think this may be causing the additional JS errors

Reply:
Yes, if the context is duplicated it seems that yes, this change is the cause.

Please remove [5a06c24](https://github.com/liferay-frontend/liferay-portal/pull/1944/commits/5a06c24a7449d5f170418acbaa2240b5910729f4)